zone/corpse.h

```diff
@@ -73,6 +73,7 @@ class Corpse : public Mob {
 	uint32			SetCharID(uint32 iCharID)	{ if (IsPlayerCorpse()) { return (char_id = iCharID); } return 0xFFFFFFFF; };
 	uint32			GetDecayTime()				{ if (!corpse_decay_timer.Enabled()) return 0xFFFFFFFF; else return corpse_decay_timer.GetRemainingTime(); }
 	uint32			GetRezTime()				{ if (!corpse_rez_timer.Enabled()) return 0; else return corpse_rez_timer.GetRemainingTime(); }
+	void            ResetDecayTimer();
```

Comment:
Formatting.

Reply:
The code above it is incorrectly using tabs for alignment, so it will never look right for different tab width settings. I aligned it to match a default tab width of 4

Reply:
No worries. Wasn’t sure. Just looked weird.